cpp/example_code/cross-service/serverless-aurora/RDSDataHandler.cpp

```diff
@@ -0,0 +1,558 @@
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+/**
+ *  RDSDataHandler.h/.cpp
+ *
+ *  The code in these 2 file implements the creation, updating and querying of a table in
+ *  an Amazon Relational Database Service (Amazon RDS).
```

Comment:
*  The code in these two files creates, updates, and queries an Amazon Relational Database Service (Amazon RDS) table.